src/table.js

```diff
@@ -343,17 +343,39 @@ export const cloneRowAt = rowIndex => tr => {
       const cloneRow = tableNode.child(rowIndex);
       // Re-create the same nodes with same attrs, dropping the node content.
       let cells = [];
+      let rowWidth = 0;
       cloneRow.forEach(cell => {
-        cells.push(
-          tableNodes[cell.type.spec.tableRole].createAndFill(
-            cell.attrs,
-            cell.marks
-          )
-        );
+        // If we're copying a row with rowspan somewhere, we dont want to copy that cell
+        // We'll increment its span below.
+        if (cell.attrs.rowspan === 1) {
+          rowWidth += cell.attrs.colspan;
```

Comment:
Correct me if I'm wrong, but you're checking the summed colspan of the previous row is < the width of the table, and if so, then adding a rowspan to any with a non-default rowspan?

Why do we care about the colspan?